services/galley/src/Galley/API/Create.hs

```diff
@@ -327,6 +327,8 @@ notifyCreatedConversation dtime usr conn c = do
   localDomain <- viewFederationDomain
   now <- maybe (liftIO getCurrentTime) pure dtime
   pushSome =<< mapM (toPush localDomain now) (Data.convLocalMembers c)
+  -- Notify remote users of being added to a conversation
+  updateRemoteConversationMemberships [] usr now c [] (Data.convRemoteMembers c)
```

Comment:
This doesn't look right. On the remote end, users will get a `MemberJoin` notification, instead of the expected `ConvCreate`. I think you should either make a new RPC for the new conversation case, or generalise this to be able to send the other kind of event.